public/app/features/alerting/unified/Analytics.ts

```diff
@@ -237,14 +237,6 @@ export function trackRulesSearchComponentInteraction(filter: keyof RulesFilter)
 export function trackRulesListViewChange(payload: { view: string }) {
   reportInteraction('grafana_alerting_rules_list_mode', { ...payload });
 }
-export function trackSwitchToSimplifiedRouting() {
```

Comment:
need to revert this, still have to figure out where to track this (preferably not in the hooks as this isn't business logic)

Reply:
I've decided to chuck this as we're not using these numbers for anything anyway